systemtests/tests/restore-test/testrunner-restore-file

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# restore some single files from archives
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+file1="$(pwd)/tmp/data/weird-files/socket.pl"
+file2="$(pwd)/tmp/data/weird-files/softlinkee"
+file3="$(pwd)/tmp/data/weird-files/softlink-to-softlinkee"
+for f in "$file1" "$file2" "$file3" ; do
+    if [[ -f "$tmp/bareos-restores/$f" ]]; then
+	rm "$tmp/bareos-restores/$f"
+    fi;
+done
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out /dev/null
+setdebug level=100 storage=File
+@$out /tmp/log1.out
+status director
+status client
+status storage=File
+wait
+messages
+@$out $tmp/jobs.out
+list jobs
+@$out $tmp/should-work.out
```

Comment:
should be unique and testname based

Reply:
Do you mean the "jobs.out" filename ?  Are the "remove or refactor" comments also refering to this ?

Reply:
jobs.out need to be the same as you check it afterward, I mean the should-work.out which is used by several tests, but actually no cleanup is done, so you don't know which tests has written that file when you use egrep

Reply:
the remove or refactor is the check for first job is a Backup. we should check it once for all, or if the check needs to be repeated for each testrunner, then it need to be refactored.